functions/date-time-offset/1.0/index.js

```diff
@@ -0,0 +1,83 @@
+import addSeconds from 'date-fns/addSeconds';
+import addHours from 'date-fns/addHours';
+import addMinutes from 'date-fns/addMinutes';
+import addDays from 'date-fns/addDays';
+import addWeeks from 'date-fns/addWeeks';
+import addMonths from 'date-fns/addMonths';
+import addYears from 'date-fns/addYears';
+import { format, parseISO } from 'date-fns';
+
+const calculateDateTimeOffset = (startDate, offset, offsetType, resultType) => {
+  let calculatedOffset = 'Invalid Date';
```

Comment:
Make this the return if the other switches are not executed. If so you can remove this variable